### PR TITLE
Ruvseq add normcounts

### PR DESCRIPTION
**Pull request by NairaNaouar**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

Changes in ruvseq wrapper and script to add Ruvseq Normalized counts output

Be aware that this normalized counts CANNOT be utilized for statistical purposes and further DESeq2, edgeR or limma analysis.
It can ONLY be used for vizualisation purposes

### CODE REVIEW

**Comment by drosofff** [comment]
@NairaNaouar there is a few R code lines to lint:
```
tools/ruvseq/ruvseq.R:3:32 Use FALSE instead of the symbol F. ( options(show.error.messages = F, error = function() { )
tools/ruvseq/ruvseq.R:160:22 Remove spaces before the left parenthesis in a function call. ( ruv_ncounts <- ifelse (is.null(opt$ruv_ncounts), FALSE, TRUE) )
tools/ruvseq/ruvseq.R:191:5 Place a space before left parenthesis, except in a function call. (   if(ruv_ncounts) { )
```

File: tools/ruvseq/ruvseq.xml
Comment (mvdbeek) [inline tools/ruvseq/ruvseq.xml:1]: Can you increase the wrapper version to 1 on line 9 ?

Comment (NairaNaouar) [reply tools/ruvseq/ruvseq.xml:1]: It's done @mvdbeek

**Comment by bgruening** [comment]
Thanks @NairaNaouar!

**Pull request merged**